docs/gitlab.md

```diff
@@ -0,0 +1,69 @@
+# Gitlab
+## Add gitlab ssh key
+
+We use **ssh** protocol for project clone on gitlab.   
+This is documentation for add ssh key on gitlab account. You can found gitlab doc [here](https://docs.gitlab.com/ee/user/ssh.html).
```

Comment:
This is a step by step guide on how to setup your ssh key on gitlab account.
You can also check the gitlab doc [here](https://docs.gitlab.com/ee/user/ssh.html).